docs/_documentations/mdt-eclipse-importedprojects.md

```diff
@@ -0,0 +1,192 @@
+---
```

Comment:
The name of this document, mdt-_eclipse_-importprojects.md implies this doc is for the Eclipse IDE, but this also applies to VS Code.  Is this going to be shared (or copied) between the two?

Reply:
The content for `mdt-eclipse-importprojects.md` and `mdt-vsc-importprojects.md` is identical. The reason there are two topics with identical content is because I spotted some interesting behaviour with the TOC when you duplicate the same file in two places. For example, when I added the same `Using a template repository of your own` topic to both Eclipse and VS Code sections, when you clicked one instance, it opened both instances which is visually challenging. Solution is have separate files for each section, even though the content's the same.